tools/exporters/data_dictionary_builder.php

```diff
@@ -138,33 +140,34 @@
 $parameterCount =0;
 
 foreach ($instruments AS $instrument) {
-    $catId ="";
+    $catId = "";
+    $table = "";
     $items =explode("\n", trim($instrument));
     foreach ($items AS $item) {
         $paramId ="";
         $bits    =explode("{@}", trim($item));
         switch($bits[0]){
         case "table":
-            $table =$bits[1];
-            print "At $table\n";
+            $table = $bits[1];
+            print "Instrument: $table\n";
             break;
 
         case "title":
-            $title  = $bits[1];
-            $error  = $DB->insert(
+            $title = $bits[1];
+            $error = $DB->insert(
                 "parameter_type_category",
                 array(
-                 'Name' => $title,
-                 'Type' => 'Instrument',
+                 "Name" => $title,
```

Comment:
are single or double quote the standard to follow in LORIS?

Reply:
I dont think we have a standard but I was getting dizzy reading through this file I decided to just change them

Reply:
Double quotes unless single quotes makes it cleaner.